tests/queries/0_stateless/02725_parquet_preserve_order.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+
+# This file has a row group with 2 rows, then a row group with 1 row.
+# It'll be read into two blocks. The first block will sleep 2x longer than the second.
+
+$CLICKHOUSE_LOCAL -q "select number+sleepEachRow(3) from file('$CURDIR/data_parquet/02725_data.parquet') settings input_format_parquet_preserve_order=1"
+$CLICKHOUSE_LOCAL -q "select number+sleepEachRow(3) from file('$CURDIR/data_parquet/02725_data.parquet') settings input_format_parquet_preserve_order=0, parallelize_output_from_storages=1"
```

Comment:
> There is no guarantee in which order blocks will be processed in this query

You mean only the second query, right?

The `sleepEachRow()` makes the non-order-preserving pipeline very likely to reorder the two blocks, because it'll sleep for 6s on the first block and 3s on the second.

The first query is the actual test, the second query checks that the test setup works, i.e. that the blocks would get reordered if the setting didn't work.

I guess your point is that relying on timing like this is too flaky. I agree, and `explain pipeline` sounds like a good idea, will switch to that.